update to latest zebra

AloeareV opened the pull request:
This fixes #355 when the zebrad binary being used is built from the commit pointed at by the Cargo.toml 3b5cbf845146ffe43be2d40b3ed5e2e6ffb7f12c (latest main as of this commit)

Let's see what CI makes of it. This _might_ break everything if pointed at pre-breaking-change zebra.

AloeareV committed:
update to latest zebra

idky137 reviewed (dismissed):
All tests pass for me locally with zebra main. 

I have approved but not merged as I am unsure if anything needs to be checked for CI.

I am also confused about the 6.1..

zancas commented on integration-tests/tests/fetch_service.rs:
```diff
@@ -63,6 +63,8 @@ async fn create_test_manager_and_fetch_service(
                 canopy: Some(1),
                 nu5: Some(1),
                 nu6: Some(1),
+                // TODO: What is network upgrade 6.1? What does a minor version NU mean?
```
It adds the lockbox.

AloeareV committed:
update comment

zancas committed:
Merge branch 'dev' into update_to_latest_zebra

nachog00 committed:
Update CI pipeline with new Zebra version
- Changed `ZEBRA_VERSION` in `.env.ci-dep-versions` to `3b5cbf8`
- Updated image tag in `.github/workflows/ci.yml` to `RUST_1.87-ZCASH_6.2.0-ZEBRA_3b5cbf8` for all jobs

The ci workflow will use the 'latest' tag normally, and should be changed back to that before merging, but setting it this way allows us to have this branch tested in ci without affecting other branches.

nachog00 commented:
This is passing all tests now. I consider it mergeable from the CI aspect.

The fix I applied is still manual. Upcoming PR's will aoutmate the process

zancas commented:
Looks like it just needs an approving review?!
